LuckyDragon/app/src/main/java/com/example/luckydragon/Models/Event.java

```diff
@@ -106,6 +106,11 @@ public String toString12h() {
 
     private List<Location> waitlistLocations = new ArrayList<>();
 
+    private ArrayList<User> waitlistUsers = new ArrayList<>();
```

Comment:
This code was removed in Matthew's PR, it shouldn't be here. I think your PR is still a bit messed up, it should only change stuff in the test files right?

Reply:
Should be fixed